docs/introduction.rst

```diff
@@ -13,92 +13,96 @@ or modifying files produced by other programs (e.g.,
+SLiM simulates diploid organisms, so each *individual* usually has two nodes;
 many operations you might want to do involve first finding the individuals you want,
 and then looking at their nodes.
 
 *******************************************
 What does SLiM record in the tree sequence?
 *******************************************
 
-Suppose we've run a very small simulation with SLiM.
-The genealogical relationships between the various diploid individuals
-who were alive over the course of the simulation might look something like
-the picture on the left below.
-Note that individuals are diploid (each has two chromosomes, or *nodes*),
-and that genealogical relationships are between the *nodes*, not the individuals.
-However, after we're done we are mainly interested in the genealogies
-of the *final* generation, and so as SLiM goes along, it *simplifies* the tree sequence,
-retaining only information necessary to describe the genealogies
-of certain individuals (the current generation and any that have been *Remembered* along the way).
-At the end, we get a tree sequence that looks something like
-the situation on the right,
-in which many of the nodes and individuals have been removed.
+Suppose we've run a very small simulation with SLiM. The genetic relationships between
+the various diploid individuals who were alive over the course of the simulation might
+look something like the picture on the left below. Note that individuals (circles) are
+diploid, so that each contains two chromosomes or *nodes* (shaded rectangles),
+and that relationships are between the *nodes*, not the individuals.
+
+At the end of the simulation we are typically only interested in the genetic
+relationships between the nodes in those individuals which are still alive;
+other parts of the genealogy are irrelevant. To save having to store this unnecessary
+genealogical information, SLiM *simplifies* the tree sequence as it goes along, retaining
+only certain parts of the genetic genealogy. When the tree sequence is output, the result
+then looks something like the situation on the right, in which many of the nodes and
+individuals have been removed.
 
 .. figure:: _static/pedigree01.png
    :scale: 40%
 
+.. _sec_left_in_tree_sequence:
 
 *************************************
 Who and what is in the tree sequence?
 *************************************
 
-OK, who and what exactly *is* left in the tree sequence?
-We clearly want all individuals of the *final* generation
-to have their genealogies recorded in the tree sequence.
-in the picture above there is one additional individuals pictured
-in the tree sequence that was not alive at the end of the simulation,
-because it was *remembered*, using the ``treeSeqRememberIndividuals()`` method.
-
+OK, who and what exactly *is* left in the tree sequence after the unnecessary
+information has been removed? Here's the diagram above with some added terminology.
 
 .. figure:: _static/pedigree2.png
```

Comment:
Hm, there's "retained" and "remembered" individuals in the diagram, but not mentioned in the text. Oh:it says "all ... dead individuals have vanished", contradicting this - maybe you meant to put a different diagram here?